tests/integration_tests/test_examples.py

```diff
@@ -304,3 +306,29 @@ def test_rl2_ml10_meta_test():
         '--episode_per_task', '1', '--meta_batch_size', '10'
     ],
                           check=False).returncode == 0
+
+
+@pytest.mark.mujoco
```

Comment:
Honestly, I'm not sure we should add *more* slow tests to the mujoco test run, but we can add these for now.